src/develop/noise_generator.h

```diff
@@ -0,0 +1,253 @@
+/*
+   This file is part of darktable,
+   Copyright (C) 2020 - Aurélien Pierre.
```

Comment:
you are reintroducing "deprecated" copyright.

Reply:
Wasn't the decision to reintroduce individual copyright ? I don't mind much either way.

Reply:
No real decision was made, Ulrich was supposed to revert this part and have proper copyright notice. At this stage I suppose we want to be consistent.

Reply:
No problem.